python/ray/util/sgd/v2/backends/backend.py

```diff
@@ -157,6 +245,14 @@ def latest_checkpoint_path(self) -> Optional[Path]:
         else:
```

Comment:
Can we consolidate the implementation for `latest_checkpoint_path` and `best_checkpoint_path`? Currently best_checkpoint_path uses the PersistedCheckpoint object to get the path, but latest_checkpoint_path does path manipulation to get the path given the directory and file name. I think we can have both of these use the path stored in the `PersistedCheckpoint` object.

Reply:
Actually do we even need `latest_checkpoint_path` anymore?